### ISSUE TITLE
Move code to oxyplot-contrib

### ISSUE THREAD

**Issue by objorke** [oxyplot/oxyplot#1400]
## Annotations
- any?

## Axes
- any?

## Series
- PieSeries
- OldCandleStickSeries
- all FinancialSeries?

**Comment by VisualMelon** [comment]
`DateTimeAxis` would seem too important to evict; are there particular issues with it that need addressing?

**Comment by objorke** [comment]
I agree that `DateTimeAxis` is important to have, striking it out now. But I have a feeling that there is a lot of issues related to it and there should probably be a re-design on how it works.

**Comment by objorke** [comment]
`FunctionAxis` is very useful in simple examples and has been very stable. I suggest to keep it.

**Comment by VisualMelon** [comment]
Regarding redesigning how DateTimeAxis works, I have been meaning for ages to implement a mechanism so that you can swap out the tick/label placing logic (i.e. put it behind an interface). This should (hopefully) allow for relatively easy implementation of things like #1241.

It would also be nice to be able to assign the label directly (i.e. rather than indicating where the tick is and letting the formatters deal with it), but that would be a big change. I'll put a proposal together to solicit comments at some point, or maybe I'll just finally get on and implement it.
